components/ContainerResourceLimit.vue

```diff
@@ -42,11 +42,11 @@ export default {
     } = this.value;
 
     return {
-      limitsCpu,
-      limitsMemory,
-      requestsCpu,
-      requestsMemory,
-      viewMode: _VIEW,
+      limitsCpu:      limitsCpu ? limitsCpu.replace(/([A-Z]|[a-z])/g, '') : null,
+      limitsMemory:   limitsMemory ? limitsMemory.replace(/([A-Z]|[a-z])/g, '') : null,
+      requestsCpu:    requestsCpu ? requestsCpu.replace(/([A-Z]|[a-z])/g, '') : null,
+      requestsMemory: requestsMemory ? requestsMemory.replace(/([A-Z]|[a-z])/g, '') : null,
+      viewMode:       _VIEW,
```

Comment:
These could be in any unit (bytes, megs, terabytes) if the container comes from kubectl/not us.  The value needs to be converted to the unit you're expecting . 
 utils/units parseSi() & formatSi({... minExponent: 2, maxExponent: 2, addSuffix: false, increment: /* 1000 for cpu, 1024 for memory */}) ).